tests/node-17.0/tests.js

```diff
@@ -0,0 +1,25 @@
+const fetch = require("node-fetch");
+
+/*
+    'req' variable has:
+        'headers' - object with request headers
+        'payload' - object with request body data
+        'env' - object with environment variables
+    'res' variable has:
+        'send(text, status)' - function to return text response. Status code defaults to 200
+        'json(obj, status)' - function to return JSON response. Status code defaults to 200
+    
+    If an error is thrown, a response with code 500 will be returned.
+*/
+
+module.exports = async (req, res) => {
+    const payload = JSON.parse(req.payload);
+
+    const todo = await fetch(`https://jsonplaceholder.typicode.com/todos/${payload.id ?? 1}`).then(r => r.json());
+
+    res.json({
+        isTest: true,
```

Comment:
We might want to add some env vars and user input so we can validate they are being accepted by the function as expected.

Reply:
We use ID of todo from payload, input should be tested. Ill add tests for environment variables and headers.

Reply:
👌